Geometry/CMSCommonData/test/DDFilteredViewAnalyzer.cc

```diff
@@ -9,9 +9,8 @@
 
```

Comment:
Are the changes in this file necessary? (i.e. doing more than changing the code formatting?)